service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/ActionCaseManagementOrderControllerMidEventTest.java

```diff
@@ -92,6 +92,8 @@ void shouldPostExpectedTemplateDataAndAddDocumentReferenceToOrderActionWhenGener
     private CaseDetails getCaseDetails() {
         CaseDetails caseDetails = populatedCaseDetails();
 
+        caseDetails.getData().put("dateOfIssue", LocalDate.now());
```

Comment:
we are moving away from using LocalDate(Time), please use the Time wrapper instead, it can be autowired in

Reply:
in test? is not simpler for me to just add like this?

Reply:
In this case it probably doesn't actually matter